core/include/tee/tee_cryp_provider.h

```diff
@@ -314,6 +305,15 @@ extern const struct crypto_ops crypto_ops;
 TEE_Result hash_sha256_check(const uint8_t *hash, const uint8_t *data,
 		size_t data_size);
 
+
+/* add entropy to PRNG entropy pool */
+TEE_Result crypto_rng_add_entropy(const uint8_t *inbuf, size_t len);
+
+/* to read random data from PRNG implementation	 */
```

Comment:
minor: can you capitalize first character for consistency with other comments and remove trailing spaces?